lib/rules/number-max-precision/README.md

```diff
@@ -37,3 +37,46 @@ a { top: 3.24px; }
 ```css
 @media (min-width: 3.23em) {}
 ```
+## Optional secondary options
+
+### `ignoreUnits: ["string"]`
```

Comment:
The `optionsMatches` util accepts regular expressions as well as strings. Can you update the examples to reflect this please, [like so](https://github.com/stylelint/stylelint/tree/master/lib/rules/unit-no-unknown#optional-secondary-options)? e.g. 

```js
["/^my-/", "%"]
```

```css
a { top: 3.245my-unit; }
```